installer/app/src/views/modal/cluster-delete.js.jsx

```diff
@@ -10,8 +10,11 @@ var ClusterDelete = React.createClass({
 					<header>
 						<h2>Delete cluster {this.props.clusterID}?</h2>
 					</header>
-
-					<button type="submit" style={RedBtnCSS}>Delete Cluster</button>
+					<label>
+						<div>Type the cluster ID to confirm deletion</div>
```

Comment:
And this could also be changed to `Type the cluster ID ({this.props.clusterID}) to confirm deletion`.